src/main/resources/config/liquibase/changelog/20220406092210_changelog.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="anar" id="20220406092210">
```

Comment:
This file is quite inconsistent. You sometimes use upper case, sometimes lower case, and once you simply capitalize the type. Please fix this.